src/rccl_wrap.cc

```diff
@@ -314,6 +268,10 @@ void rcclSetPxn(struct ncclComm* comm,  int& rcclPxnDisable) {
   }
   rcclPxnDisable = pxnDisable;
   comm->enableCustColl = !pxnDisable;
+  if (comm->enableCustColl) {
+	setenv("NCCL_ALLOC_P2P_NET_LL_BUFFERS", "1", 0);
+	setenv("NCCL_P2P_LL_THRESHOLD", "1024", 0);
```

Comment:
pls add a comment next to the declarations for this environments that they maybe be overridden by changes in rcclSetPxn